roles/ceph-osd/tasks/check_mandatory_vars.yml

```diff
@@ -26,32 +26,24 @@
     - osd_group_name is defined
     - osd_group_name in group_names
     - not containerized_deployment
-    - not journal_collocation
-    - not raw_multi_journal
-    - not dmcrypt_journal_collocation
-    - not dmcrypt_dedicated_journal
+    - osd_scenario != 'dummy'
 
-- name: verify only one osd scenario was chosen
+- name: make sure a valid osd scenario was chosen
   fail:
-    msg: "please select only one osd scenario"
+    msg: "please choose an osd scenario"
   when:
     - osd_group_name is defined
     - osd_group_name in group_names
     - not containerized_deployment
-    - (journal_collocation and raw_multi_journal)
-      or (dmcrypt_journal_collocation and journal_collocation)
-      or (dmcrypt_journal_collocation and raw_multi_journal)
-      or (dmcrypt_dedicated_journal and journal_collocation)
-      or (dmcrypt_dedicated_journal and raw_multi_journal)
-      or (dmcrypt_dedicated_journal and dmcrypt_journal_collocation)
+    - not osd_scenario in ['journal_collocation', 'raw_multi_journal', 'dmcrypt_dedicated_journal', 'dmcrypt_journal_collocation']
```

Comment:
I'm thinking we should keep this list in ``defaults/main.yml``. Then we could also print the list as part of the failure message here.